docs/llm/quickstart.md

```diff
@@ -4,25 +4,27 @@ Let's deploy [OpenVINO/Phi-3.5-mini-instruct-int4-ov](https://huggingface.co/Ope
 It is [microsoft/Phi-3.5-mini-instruct](https://huggingface.co/microsoft/Phi-3.5-mini-instruct) quantized to INT4 precision and converted to IR format.
 
 ## Requirements
-- Linux or Windows11
+- Linux or Windows 11
 - Docker Engine or `ovms` binary package [installed](../deploying_server_baremetal.md)
 - Intel iGPU or ARC GPU 
 
 ## Deployment Steps
 
 ### 1. Install Python dependencies:
-```console
-pip3 install huggingface_hub jinj2
+```bash
+pip3 install huggingface_hub jinja2
 ```
 
 ### 2. Download and Prepare the Model:
 Using `export_model.py` script, download the OpenVINO model and prepare models repository including all configuration required for deployment with OpenVINO Model Server. For details, see [Exporting GEN AI Models](../../demos/common/export_models/README.md).
 
-```console
+```bash
```

Comment:
our automated tests for documentation is using `console` tag as an indicator that command can be run in windows cmd and also in bash. The tag `bash` mean the command is applicable for linux only. For windows only command we use `bat`